core/src/main/java/jenkins/security/csrf/CSRFAdministrativeMonitor.java

```diff
@@ -0,0 +1,37 @@
+package jenkins.security.csrf;
+
+import hudson.Extension;
+import hudson.model.AdministrativeMonitor;
+import hudson.security.csrf.CrumbIssuer;
+import jenkins.model.Jenkins;
+import org.jenkinsci.Symbol;
+import org.kohsuke.accmod.Restricted;
+import org.kohsuke.accmod.restrictions.NoExternalUse;
+
+/**
+ * Monitor that the CSRF is enabled on the application.
+ *
+ * @author Wadeck Follonier
+ * @author Oleg Nenashev
+ * @since TODO
+ */
+@Extension
+@Symbol("csrf")
```

Comment:
Are we still maintaining these? Why? Isn't https://github.com/jenkinsci/system-config-dsl-plugin dead?

Reply:
It's not dead. @ewelinawilkosz2 and @Andrey9kin adopted it.
https://github.com/Praqma/system-config-dsl-plugin

So I would keep symbol annotations when possible. CC @ndeloof

Reply:
Oh, that's nice. I was unaware this is `system-config-dsl` based.

Reply:
Agree. We should have @symbol mostly everywhere on components which end user might need to access for configuration, as sort of a 'natural' namr

Reply:
Just to be clear, we're now working on https://github.com/Praqma/jenkins-coco-plugin which is `system-config-dsl-plugin` based but with a new way of doing some things. As soon as we decide this is how we want to keep things we'll try to announce it using known channels